src/promptflow/setup.py

```diff
@@ -98,11 +98,15 @@
             # MDC dependencies for monitoring
             "azureml-ai-monitoring>=0.1.0b3,<1.0.0",
         ],
+        "executor-service": [
+            "uvicorn>=0.27.0",  # used to start executor service
+            "fastapi>=0.109.0",  # used to build web executor server
```

Comment:
please also add a upper range for dependencies

Reply:
Added.